### PR TITLE
Add com.krillbite.ThePlan

### PR DESCRIPTION
**Pull request by FakeShemp**
http://www.krillbite.com/theplan/

Still not sure how to solve one archive having all arches' executables in a nice way.

### CODE REVIEW

**Comment by nedrichards** [comment]
bot, build com.krillbite.ThePlan

File: com.krillbite.ThePlan.desktop [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[Desktop Entry]
+Type=Application
+Version=1.3
+Name=The Plan
+Exec=ThePlan.sh
+Icon=com.krillbite.ThePlan
+Categories=Game;AdventureGame
+Name[en_US]=com.krillbite.ThePlan.desktop
```
Comment (TingPing) [inline com.krillbite.ThePlan.desktop]: That is certainly the wrong name.

File: com.krillbite.ThePlan.yaml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+runtime: org.freedesktop.Platform
+runtime-version: 1.6
+sdk: org.freedesktop.Sdk
+command: ThePlan.sh
+finish-args:
+- --share=ipc
+- --socket=x11
+- --device=dri
+- --socket=pulseaudio
+- --filesystem=~/.config/unity3d:create
```
Comment (TingPing) [inline com.krillbite.ThePlan.yaml]: Why does this need to be on the host?

File: com.krillbite.ThePlan.yaml [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+  - install -Dm755 'The Plan.x86_64' '/app/The Plan.x86_64'
+  - install -Dm755 launcher.sh /app/bin/ThePlan.sh
+  - install -Dm644 com.krillbite.ThePlan.appdata.xml /app/share/appdata/com.krillbite.ThePlan.appdata.xml
+  - install -Dm644 com.krillbite.ThePlan.desktop /app/share/applications/com.krillbite.ThePlan.desktop
+  - install -Dm644 icon32x32.png /app/share/icons/hicolor/32x32/apps/com.krillbite.ThePlan.png
+  - install -Dm644 icon64x64.png /app/share/icons/hicolor/64x64/apps/com.krillbite.ThePlan.png
+  - install -Dm644 icon128x128.png /app/share/icons/hicolor/128x128/apps/com.krillbite.ThePlan.png
+
+  sources:
+  - type: archive
```
Comment (TingPing) [inline com.krillbite.ThePlan.yaml]: This is proprietary so we can't redistribute it, you need an `extra-data` source type, see other proprietary apps like Discord.

**Comment by nedrichards** [comment]
bot, build com.krillbite.ThePlan

**Comment by nedrichards** [comment]
bot, build com.krillbite.ThePlan

File: com.krillbite.ThePlan.yaml [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+sdk: org.freedesktop.Sdk
+command: ThePlan.sh
+tags:
+  - proprietary
+finish-args:
+- --share=ipc
+- --socket=x11
+- --device=dri
+- --socket=pulseaudio
+- --persist=.config/unity3d
```
Comment (barthalion) [inline com.krillbite.ThePlan.yaml]: Here, actually, it probably respects `$XDG_CONFIG_HOME` env var so can be skipped completely.

**Comment by nedrichards** [comment]
bot, build com.krillbite.ThePlan

File: com.krillbite.ThePlan.yaml (hunk 4 above)
Comment (FakeShemp) [reply com.krillbite.ThePlan.yaml]: Unfortunately, Unity 4, which this game is built in, doesn't. It's hard-coded to .config/unity3d and disregards xdg variables.

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/com.krillbite.ThePlan

**Pull request closed without merging**